diff --git a/crates/diesel_models/src/schema.rs b/crates/diesel_models/src/schema.rs
index b35e809a0..749a5dd47 100644
--- a/crates/diesel_models/src/schema.rs
+++ b/crates/diesel_models/src/schema.rs
@@ -221,8 +221,30 @@
     use crate::spanner_types::HsTimestamp as Timestamp;
     use crate::spanner_types::HsJson as Json;
 
-    blocklist_fingerprint (id) {
-        id -> Int8,
+    blocklist (merchant_id, fingerprint_id) {
+        #[max_length = 64]
+        merchant_id -> Varchar,
+        #[max_length = 64]
+        fingerprint_id -> Varchar,
+        data_kind -> BlocklistDataKind,
+        metadata -> Nullable<Jsonb>,
+        created_at -> Timestamp,
+        #[max_length = 64]
+        processor_merchant_id -> Nullable<Varchar>,
+        #[max_length = 255]
+        created_by -> Nullable<Varchar>,
+        #[max_length = 64]
+        profile_id -> Nullable<Varchar>,
+    }
+}
+
+diesel::table! {
+    use diesel::sql_types::*;
+    use crate::enums::diesel_exports::*;
+    use crate::spanner_types::HsTimestamp as Timestamp;
+    use crate::spanner_types::HsJson as Json;
+
+    blocklist_fingerprint (merchant_id, fingerprint_id) {
         #[max_length = 64]
         merchant_id -> Varchar,
         #[max_length = 64]
@@ -239,8 +261,7 @@
     use crate::spanner_types::HsTimestamp as Timestamp;
     use crate::spanner_types::HsJson as Json;
 
-    blocklist_lookup (id) {
-        id -> Int8,
+    blocklist_lookup (merchant_id, fingerprint) {
         #[max_length = 64]
         merchant_id -> Varchar,
         fingerprint -> Text,
@@ -465,7 +486,6 @@
     use crate::spanner_types::HsJson as Json;
 
     configs (key) {
-        id -> Int8,
         #[max_length = 255]
         key -> Varchar,
         config -> Text,
@@ -881,8 +901,7 @@
     use crate::spanner_types::HsTimestamp as Timestamp;
     use crate::spanner_types::HsJson as Json;
 
-    locker_mock_up (id) {
-        id -> Int8,
+    locker_mock_up (card_id) {
         #[max_length = 255]
         card_id -> Varchar,
         #[max_length = 255]
@@ -2036,6 +2055,7 @@
     api_keys,
     authentication,
     batch_blocklist_jobs,
+    blocklist,
     blocklist_fingerprint,
     blocklist_lookup,
     business_profile,
