diff --git a/crates/diesel_models/src/schema.rs b/crates/diesel_models/src/schema.rs
--- a/crates/diesel_models/src/schema.rs
+++ b/crates/diesel_models/src/schema.rs
@@ -211,8 +211,28 @@
     use diesel::sql_types::*;
     use crate::enums::diesel_exports::*;
 
-    blocklist_fingerprint (id) {
-        id -> Int4,
+    blocklist (merchant_id, fingerprint_id) {
+        #[max_length = 64]
+        merchant_id -> Varchar,
+        #[max_length = 64]
+        fingerprint_id -> Varchar,
+        data_kind -> BlocklistDataKind,
+        metadata -> Nullable<Jsonb>,
+        created_at -> Timestamp,
+        #[max_length = 64]
+        processor_merchant_id -> Nullable<Varchar>,
+        #[max_length = 255]
+        created_by -> Nullable<Varchar>,
+        #[max_length = 64]
+        profile_id -> Nullable<Varchar>,
+    }
+}
+
+diesel::table! {
+    use diesel::sql_types::*;
+    use crate::enums::diesel_exports::*;
+
+    blocklist_fingerprint (merchant_id, fingerprint_id) {
         #[max_length = 64]
         merchant_id -> Varchar,
         #[max_length = 64]
@@ -227,8 +247,7 @@
     use diesel::sql_types::*;
     use crate::enums::diesel_exports::*;
 
-    blocklist_lookup (id) {
-        id -> Int4,
+    blocklist_lookup (merchant_id, fingerprint) {
         #[max_length = 64]
         merchant_id -> Varchar,
         fingerprint -> Text,
@@ -440,7 +459,6 @@
     use crate::enums::diesel_exports::*;
 
     configs (key) {
-        id -> Int4,
         #[max_length = 255]
         key -> Varchar,
         config -> Text,
@@ -888,8 +906,7 @@
     use diesel::sql_types::*;
     use crate::enums::diesel_exports::*;
 
-    locker_mock_up (id) {
-        id -> Int4,
+    locker_mock_up (card_id) {
         #[max_length = 255]
         card_id -> Varchar,
         #[max_length = 255]
@@ -1982,6 +1999,7 @@
     api_keys,
     authentication,
     batch_blocklist_jobs,
+    blocklist,
     blocklist_fingerprint,
     blocklist_lookup,
     business_profile,
